info.colobot.Colobot.yaml

```diff
@@ -0,0 +1,179 @@
+---
+app-id: info.colobot.Colobot
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+command: colobot
+rename-icon: colobot
+rename-desktop-file: colobot.desktop
+finish-args:
+  - "--share=ipc"
+  - "--socket=x11"
+  - "--device=dri"
+  - "--socket=pulseaudio"
+  - "--env=LD_LIBRARY_PATH=/app/lib64:/app/lib"
+build-options:
+  cflags: "-O2 -g"
+  cxxflags: "-O2 -g"
+cleanup:
+  - "/include"
+  - "/lib/pkgconfig"
+  - "/share/pkgconfig"
+  - "/share/aclocal"
+  - "/share/doc"
+  - "/man"
+  - "/share/man"
+  - "/share/gtk-doc"
+  - "/share/vala"
+  - "*.la"
+  - "*.a"
+modules:
+  - shared-modules/glew/glew.json
+  - shared-modules/glu/glu-9.0.0.json
+
+  - name: boost
+    buildsystem: simple
+    build-commands:
+      - "./bootstrap.sh --prefix=/app --with-libraries=system,filesystem,regex"
+      - "./b2 headers"
+      - "./b2 install"
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/project/boost/boost/1.64.0/boost_1_64_0.tar.bz2
+        sha256: 7bcc5caace97baa948931d712ea5f37038dbb1c5d89b43ad4def4ed7cb683332
+
+  - name: physfs
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DPHYSFS_BUILD_TEST=off
+    sources:
+      - type: archive
+        url: https://icculus.org/physfs/downloads/physfs-3.0.1.tar.bz2
+        sha256: b77b9f853168d9636a44f75fca372b363106f52d789d18a2f776397bf117f2f1
+
+  # Provided libsndfile is compiled without ogg/vorbis support
+  - name: sndfile
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DBUILD_STATIC_LIBS=off
+      - -DBUILD_SHARED_LIBS=on
+      - -DBUILD_PROGRAMS=off
+      - -DBUILD_EXAMPLES=off
+      - -DBUILD_TESTING=off
+      - -DENABLE_CPACK=off
+    sources:
+      - type: archive
+        url: https://github.com/erikd/libsndfile/archive/b1194240eae381c380797e400e3ef490f73be3ab.zip # Until 1.0.29 is released
+        sha256: a662ba291aa575ec6656473db2a417d5df4b98780e32d556f160bc7d55da656d
+
+  - name: colobot
+    builddir: true
+    buildsystem: cmake-ninja
+    build-options:
+      cxxflags: "-O2 -g -Wno-format-overflow"
+    config-opts:
+      - -DCMAKE_LIBRARY_PATH=/app/lib64
```

Comment:
Wrong dir, can probably be removed entirely.

Reply:
Only there because of the libsndfile requirement - and a small bug in their CMake lists, so if it's fixed in the SDK/runtime then all /app/lib(64) usage will disappear.